frontend/src/components/SkillGroup/SkillItem/SkillItemContainer.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC, useState } from 'react';
+
+import SkillItem from './SkillItem';
+import { ISkillItemContainerProps } from '../types';
+
+const SkillItemContainer: FC<ISkillItemContainerProps> = ({ name, skillImage }) => {
+  const [isSkillNameShown, setSkillNameShown] = useState(false);
+
+  const showSkillName = () => {
+    setSkillNameShown(true);
+  };
+
+  const hideSkillName = () => {
+    setSkillNameShown(false);
+  };
```

Comment:
Do we really need two of these functions? Can we pass "setSkillNameShown" to the component at once and thus replace two props with one? What do you think?